codesign_policy.json

```diff
@@ -0,0 +1,125 @@
+{
+  "Version": "1.0.0",
+  "UseMinimatch": true,
+  "SignBatches": [
+    {
+      "MatchedPath": [
+        "Microsoft.DocAsCode*.dll",
+        "*.exe"
+      ],
+      "SigningInfo": {
+        "Operations": [
+          {
+            "KeyCode": "CP-230012",
+            "OperationSetCode": "SigntoolSign",
+            "parameters": [
+              {
+                "parameterName": "OpusName",
+                "parameterValue": "Microsoft"
+              },
+              {
+                "parameterName": "OpusInfo",
+                "parameterValue": "http://www.microsoft.com"
+              },
+              {
+                "parameterName": "PageHash",
+                "parameterValue": "/NPH"
+              },
+              {
+                "parameterName": "TimeStamp",
+                "parameterValue": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
+              },
+              {
+                "parameterName": "FileDigest",
+                "parameterValue": "/fd \"SHA256\""
+              }
+            ],
+            "ToolName": "sign",
+            "ToolVersion": "1.0"
+          },
+          {
+            "KeyCode": "CP-230012",
+            "OperationSetCode": "SigntoolVerify",
+            "Parameters": [
+              {
+                "parameterName": "VerifyAll",
+                "parameterValue": "/all"
+              }
+            ],
+            "ToolName": "sign",
+            "ToolVersion": "1.0"
+          }
+        ]
+      }
+    },
+    {
+      "MatchedPath": [
+          "DotLiquid.resources.dll",
```

Comment:
I think you used `!Microsoft*.dll` in this PR earlier. Can it work?

Reply:
Doesn't seem to work, therefore explicitly listed them.